frontend/event/eventDetailsDirective.js

```diff
@@ -148,6 +148,28 @@
             return new Date(isoTime).getHours();
         };
 
+        eventCtrl.isFollower = () => {
+            return eventCtrl.event && _.includes(eventCtrl.event.followers, eventCtrl.user.key);
```

Comment:
there's no need to use lodash 
`eventCtrl.event.followers.includes(eventCtrl.user.key)`